.coveragerc

```diff
@@ -0,0 +1,12 @@
+[run]
+data_file = .coverage_mqt  # to avoid being overridden by server tests
+omit = *system_site_packages*
+
+[report]
+# Regexes for lines to exclude from consideration
+exclude_lines =
+    # Have to re-enable the standard pragma
+    pragma: no cover
+
+    # Don't complain if non-runnable code isn't run:
+    if __name__ == .__main__.:
```

Comment:
@bwrsandman because the content of the `if` block you exclude is not matched by the regexp? (only the `if` line itself)